bindings/rust/wasmedge-sys/README.md

```diff
@@ -6,21 +6,6 @@ For developers, it is recommended that the APIs in `wasmedge-sys` are used to co
 
 * Notice that [wasmedge-sys](https://crates.io/crates/wasmedge-sys) requires **Rust v1.66 or above** in the **stable** channel.
 
-## Versioning Table
```

Comment:
Let's maintain this table. Why do you remove it?

Reply:
The versioning table is still there. Just move it into a new section of this file. Thanks!

Reply:
We moved the versioning table into the `Build` section in the previous PR, and in that PR we forgot to remove the old one. Thanks.